components/NavigationBar.js

```diff
@@ -19,11 +20,12 @@ export default function NavigationBar () {
   )
 }
 
-function NavigationLink ({ href, disabled, children }) {
+function NavigationLink({ href, disabled, children }) {
   return (
-    <span className={disabled ? 'disabled' : ''}>
-      <LevitatingLink href={href}>{children}</LevitatingLink>
-      <style jsx>{`
+    <Link to={children.toLowerCase()} smooth={true} duration={500} >
+      <span className={disabled ? 'disabled' : ''}>
+        <LevitatingLink href={href}>{children}</LevitatingLink>
```

Comment:
Also, the smooth scrolling doesn’t seem to work, and the disabled items is not displaying properly. Here is a screenshot in Google Chrome.

![image](https://cloud.githubusercontent.com/assets/193136/26275740/2a9a79ae-3d92-11e7-96e6-d61aa758410c.png)